modules/project-factory/schemas/project.schema.json

```diff
@@ -108,6 +108,9 @@
     "iam_by_principals": {
       "$ref": "#/$defs/iam_by_principals"
     },
+    "iam_by_additive": {
```

Comment:
why not keep the same `iam_by_principals_additive` name?

Reply:
because it's a typo :)